core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -65,7 +65,7 @@
 				'id' => 'primary',
 				'alias' => 'index',
 				'type,dns' => 'index',
-				'pid,type,start,stop,published' => 'index'
+				'pid,type,published,start,stop' => 'index'
```

Comment:
@ausi `pid,published,type,start,stop`?

Reply:
This one definitely makes a difference. Before: key_len 252, filtered 2.56%. After: key_len 266, filtered 16%.